sdk/formrecognizer/ai-form-recognizer/samples/typescript/src/shims.d.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* eslint  @typescript-eslint/no-empty-interface: 0 */
+
+interface FileReader {
```

Comment:
Hmm. We can do something similar to what core-http does to avoid needing these shims. It would be unfortunate if everyone using our package needed a shim for FileReader.

Reply:
Yes definitely need to fix this for preview.1. https://github.com/azure/azure-sdk-for-js/issues/8420